src/screens/ChangePin.js

```diff
@@ -159,8 +159,12 @@ class ChangePin extends React.Component {
   }
 
   executeChangePin = () => {
-    const success = hathorLib.wallet.changePin(this.state.pin1, this.state.pin2);
-    if (success) {
+    const successChangePin = hathorLib.wallet.changePin(this.state.pin1, this.state.pin2);
```

Comment:
Should we create an util method `changePin` that call both methods?

Reply:
I agree, done in d8390ee0fdec2e30e0b0afbc606ab4a481c42661